packages/workspace/src/command-line/workspace-schematic.ts

```diff
@@ -135,7 +135,7 @@ function createWorkflow(dryRun: boolean) {
     root,
     dryRun,
     registry: new schema.CoreSchemaRegistry(formats.standardFormats),
-    resolvePaths: [root],
+    resolvePaths: [process.cwd(), rootDirectory, root, __dirname],
```

Comment:
- `process.cwd()` and `__dirname` were added as per [Angular Devkit's source](https://github.com/angular/angular-cli/blob/master/packages/angular/cli/models/schematic-command.ts#L257)
- `rootDirectory` should be the correct, un-normalized route
- we also send the normalized `root` (which was there before) just in case it's different from the original one